server_addon/houdini/client/ayon_houdini/plugins/load/load_hda.py

```diff
@@ -28,14 +28,18 @@ def load(self, context, name=None, namespace=None, data=None):
         # Get the root node
         obj = hou.node("/obj")
 
-        # Create a unique name
-        counter = 1
         namespace = namespace or context["folder"]["name"]
-        formatted = "{}_{}".format(namespace, name) if namespace else name
-        node_name = "{0}_{1:03d}".format(formatted, counter)
+        node_name = "{}_{}".format(namespace, name) if namespace else name
 
         hou.hda.installFile(file_path)
-        hda_node = obj.createNode(name, node_name)
+
+        # Get the type name from the HDA definition.
+        hda_defs = hou.hda.definitionsInFile(file_path)
+        if not hda_defs:
+            raise RuntimeError(f"No HDA definitions found in file: {file_path}")
```

Comment:
this should be `LoaderError` probably

Reply:
your wish is my command https://github.com/ynput/ayon-core/pull/644/commits/f8e928f36c388b2335b421477fbea38fd2832ae5